core/pkgmanager.go

```diff
@@ -90,25 +90,25 @@ func GetAurPkgCommand(command string) []string {
 	switch command {
 	// defaults
 	case "autoremove":
-		return []string{"echo", "Not implemented yet! "}
-	case "clean":
 		return []string{bin, "-Yc"}
```

Comment:
`-Yc` is not a valid command (<https://archlinux.org/pacman/pacman.8.html>), the equivalent to `autoremove` should be `pacman -Qdtq | pacman -Rs -`

Reply:
The AUR contaner uses the Yay package manager. `-Yc` is a command implemented by the Yay package manager, as shown here:
![image](https://user-images.githubusercontent.com/48618519/229819551-5574672b-045e-455d-8fb6-ec15f1088efd.png)

Reply:
Does `-Yc` include optdepends? In my opinion these should not be removed

Reply:
No, that would be `-Ycc`